### PR TITLE
New logic updates

### PR DESCRIPTION
**Pull request by ChrisMcGowan**
## Changes proposed in this pull request:

- Add the 2nd clamav-config job to allow changes from the main job set in runtime-config
- Allow more customization in the clam and freshclam files

## Security considerations

Active AV is a good thing

### CODE REVIEW

**Comment by ChrisMcGowan** [comment]
re-checking before cutting release

**Comment by ChrisMcGowan** [comment]
Re-tested a dev release and works as tested

File: jobs/clamav/templates/conf/clamd.conf.erb [hunk 1]
```diff
@@ -20,36 +20,81 @@ ScanPE false
...
 <%= opt %>
 <% end %>
 <% end %>
+<% if_link('clamav-config') do |clamavconfig| %>
+  <% clamavconfig.if_p("clamav-config.additional_options") do |addopt| %>
+<% addopt.each do |opt| %>
+<%= opt %>
+  <% end %>
+  <% end %>
+<% end %>
```
Comment (bengerman13) [inline jobs/clamav/templates/conf/clamd.conf.erb]: this reads like clamav-config.additional_options is a nested list - is that right?

Also - I think indenting lines 96 and 97 would make it easier to line up the `end`s to their `if`s and `do`s

Comment (ChrisMcGowan) [reply jobs/clamav/templates/conf/clamd.conf.erb]: In essence yes, it's nested under the if_link. but 97 needs to be there as to not introduce any spaces before the options added.  I'll run some tests

**Comment by ChrisMcGowan** [comment]
@bengerman13 fixed the intent

**Review by bengerman13 — APPROVED** [review]

**Comment by bengerman13** [comment]
LGTM.
I think at some point we'll probably want the flexibility to specify separate {in|ex}clude_directories for scheduled vs onaccess scans, but that's a future improvement

**Pull request merged**